src/main.js

```diff
@@ -204,20 +206,23 @@ function createYourOwnPoster(event) {
     quote: quoteOnPoster.innerText = motivationalQuoteInput.value
   };
   currentPoster = userCurrentPoster; 
-  console.log(imageOnPoster);
-  console.log(titleOnPoster);
-  console.log(quoteOnPoster);
   event.preventDefault();
   showMainPosterView();
-  console.log(currentPoster);
   return currentPoster;
```

Comment:
@em2396 Should we should remove all the console.logs here before merging?

Reply:
Oops! Ignore this. It already IS taken out. I looked through this too quickly. 😂